json-rpc/src/stream_rpc/logging.rs

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) The Diem Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use diem_logger::Schema;
```

Comment:
Hmm... I thought the idea was to have a single logger schema per crate and add appropriate fields as optional, see: https://github.com/diem/diem/blob/main/consensus/safety-rules/src/logging.rs#L10
The current approach would require grouping on the source path rather than the log type afaiu

Reply:
I simplified it to two only two loggers- is this acceptable or would you like me to merge these as well?

It's a bit harder to merge these into the root crate log because they call it with a macro. Can still do it though if you'd prefer.